docs/rfcs/1-command-line-design.md

```diff
@@ -0,0 +1,45 @@
+- Author: xiongjiwei <xiongjiwei1996@outlook.com>
+- Start Date: 2021-08-29
+- RFC PR: N/A
+- Tracking Issue: [beyondstorage/beyond-ftp#12](https://github.com/beyondstorage/beyond-ftp/issues/12)
+
+# RFC-1: Command Line Design
+
+## Background
+
+There are many different configs can be set in Beyond-FTP, such as storager type, host, and port. It is useful to design a command line interface to make some of them easy to configurable.
+
+## Proposal
+
+So I propose the following commands:

```

Comment:
A bit of idea before review: maybe we need to design `commands`, `args` and `flags`.

- Will beyond-ftp have subcommand?
- Will beyond-ftp have args?
- Will beyond-ftp have flags? What's the type of flag? Is the flag required? Is the flag have a default value?
- What's the config format? Can we set value from env? What's the apply order? 
- What's flag parse lib to use? How about use https://github.com/urfave/cli instead?
- Do we need to follow the normal golang application layout? Aka, put `main.go` in `cmd/beyondftp` instead of root paht.